packages/ag-charts-community/.dependency-cruiser.js

```diff
@@ -0,0 +1,443 @@
+
+        {
+            name: 'not-to-spec',
+            comment:
+                'This module depends on a spec (test) file. The sole responsibility of a spec file is to test code. ' +
+                "If there's something in a spec that's of use to other modules, it doesn't have that single " +
+                'responsibility anymore. Factor it out into (e.g.) a separate utility/ helper or a mock.',
+            severity: 'error',
+            from: {},
+            to: {
+                path: '.(spec|test).(js|mjs|cjs|ts|ls|coffee|litcoffee|coffee.md)$',
+            },
+        },
+        {
+            name: 'not-to-dev-dep',
+            severity: 'error',
+            comment:
+                "This module depends on an npm package from the 'devDependencies' section of your " +
+                'package.json. It looks like something that ships to production, though. To prevent problems ' +
+                "with npm packages that aren't there on production declare it (only!) in the 'dependencies'" +
+                'section of your package.json. If this module is development only - add it to the ' +
+                'from.pathNot re of the not-to-dev-dep rule in the dependency-cruiser configuration',
+            from: {
+                path: '^(src)',
+                pathNot: '(src/chart/test)|(.(spec|test).(js|mjs|cjs|ts|ls|coffee|litcoffee|coffee.md)$)',
+            },
+            to: {
+                dependencyTypes: ['npm-dev'],
+            },
+        },
+        {
+            name: 'optional-deps-used',
+            severity: 'info',
+            comment:
+                'This module depends on an npm package that is declared as an optional dependency ' +
+                "in your package.json. As this makes sense in limited situations only, it's flagged here. " +
+                "If you're using an optional dependency here by design - add an exception to your" +
+                'dependency-cruiser configuration.',
+            from: {},
+            to: {
+                dependencyTypes: ['npm-optional'],
+            },
+        },
+        {
+            name: 'peer-deps-used',
+            comment:
+                'This module depends on an npm package that is declared as a peer dependency ' +
+                'in your package.json. This makes sense if your package is e.g. a plugin, but in ' +
+                'other cases - maybe not so much. If the use of a peer dependency is intentional ' +
+                'add an exception to your dependency-cruiser configuration.',
+            severity: 'warn',
+            from: {},
+            to: {
+                dependencyTypes: ['npm-peer'],
+            },
+        },
+        {
+            name: 'ag-isolated-options',
+            comment: 'Options modules should be isolated from implementation modules.',
+            severity: 'error',
+            from: { path: 'src/options' },
+            to: { pathNot: '^src/options' },
+        },
```

Comment:
FYI: The rest of the config here is boilerplate, this is the section I added for our particular use-case.